runtime/lua/vim/lsp.lua

```diff
@@ -1260,8 +1260,13 @@ function lsp.start_client(config)
         changetracking.reset(client)
       end
       if code ~= 0 or (signal ~= 0 and signal ~= 15) then
-        local msg =
-          string.format('Client %s quit with exit code %s and signal %s', client_id, code, signal)
+        local msg = string.format(
+          'Client %s quit with exit code %s and signal %s. Check %s for errors',
```

Comment:
variable part of a message, especially potentially long values, should go at end if possible

```suggestion
          'Client %s quit with exit code %s and signal %s. Check log for errors: %s',
```